### ISSUE TITLE
Metal IOs support 

### ISSUE THREAD

**Issue by RobinGustav** [ColinLeung-NiloCat/UnityURPUnlitScreenSpaceDecalShader#23]
Hello,

Thank you for this wonderful shader, it work like a charm and performance are nice.

I recently try to adapt my project from android to IOs but i have some issues.
OpenGL ES 3.0 is now depreciated on Iphone so Metal is mandatory. 
On your requirement, your say ;
"For mobile, you need at least OpenGLES3.0 (#pragma target 3.0 due to ddx() & ddy())"

Metal support #pragma tartget 3.0 and newer
(source: https://docs.unity3d.com/Manual/SL-ShaderCompileTargets.html)

On Unity Mac editor everything worked perfectly,
When i compile, no errors are spotted so i thought it would work.
But in practice on the device this shader behave very strangely;
it's like the decal is at the opposite distance of the camera by symmetry on the mesh who should receive this decal.

I don't know where the problem is, my knowledges on shader are very basics so i thinks it's might be a lake of support of some features.
Have you any ideas what can create such behaviour ?

Thanks you again for your great work, the help and time that you can provide.

Have a good day

**Comment by Tommie0607** [comment]
Yes I have the same issue. I'm looking into the shader but it's not an easy one :)
